packages/components/src/FieldGroup/_docs/FieldGroup.mdx

```diff
@@ -0,0 +1,37 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as FieldGroupStories from "./FieldGroup.stories"
+
+<Meta of={FieldGroupStories} />
+
+# FieldGroup
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/FieldGroup"
+  figma=""
+  designGuidelines=""
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { FieldGroup } from "@kaizen/components"'
+/>
+
+## Overview
+
+A simple wrapper to wrap form field components and their related siblings. This a primitive is used in components like `TextField`, `TextArea`, `Slider`, etc.
+
+<Canvas of={FieldGroupStories.Playground} />
+<Controls of={FieldGroupStories.Playground} />
+
+## API
+
+## Inline
+Changes the FieldGroup to an inline element with no bottom margin. This can be seen in the `Slider` component.
```

Comment:
Link to slider docs even if it doesn't exist yet? It'll be in the top level path `Components/Slider`

Reply:
This works btw
https://github.com/cultureamp/kaizen-design-system/pull/4101/files#diff-fdd22997bb913ba2818583a1d6674cf5021e66584415e3457b5140985a759289R27

Reply:
Nice, updated so that should be good to link the slider docs page when that exists :)